application/src/main/java/org/togetherjava/tjbot/commands/tags/TagManageCommand.java

```diff
@@ -358,55 +358,45 @@ private void logAction(@NotNull Subcommand subcommand, @NotNull Guild guild,
             @NotNull User author, @NotNull TemporalAccessor triggeredAt, @NotNull String id,
             @Nullable String newContent, @Nullable String previousContent) {
 
-        if (Subcommand.SUBCOMMANDS_WITH_NEW_CONTENT.contains(subcommand) && newContent == null) {
-            throw new IllegalArgumentException(
-                    "newContent is null even though the subcommand should supply a value.");
-        }
+        ArrayList<ModAuditLogWriter.Attachment> attachments = new ArrayList<>();
```

Comment:
List, not ArrayList on the left side

Reply:
Am I allowed to ask how does it differ?

Reply:
Liscov principle. You should only demand the type u actually need

Reply:
Not a big deal in this particular case. But you should get used to it.

Reply:
Ok. changed it.